### ISSUE TITLE
[Feature]: Missing OpenTofu Registry GPG Key

### ISSUE THREAD

**Issue by metaloid2k21** [snowflakedb/terraform-provider-snowflake#2939]
### Use Cases or Problem Statement

When using snowflake-labs/snowflake provider, getting missing registry GPG key error:
```
Installing snowflake-labs/snowflake v0.84.0...
Installed snowflake-labs/snowflake v0.84.0. Signature validation was skipped due to the registry not containing GPG keys for this provider
```
Opened https://github.com/opentofu/registry/issues/727 and got response that the Provider project [should submit provider's keys.](https://github.com/opentofu/registry/issues/727#issuecomment-2228256696).

### Category

category:resource

### How much impact is this issue causing?

Medium

### Would you like to implement a fix?

- [ ] Yeah, I'll take it 😎

**Comment by sfc-gh-jmichalak** [comment]
Hi @metaloid2k21 👋 Currently, support for OpenTofu is not on our roadmap. cc @sfc-gh-asawicki

**Comment by sfc-gh-asawicki** [comment]
Hey @metaloid2k21. Currently, we only publish our provider to the [official terraform registry](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs).

**Comment by metaloid2k21** [comment]
Hello @sfc-gh-asawicki 
Thanks for your feedback.
The snowflake-labs/terraform-provider-snowflake is already available at the OpenTofu registry: https://github.com/opentofu/registry/tree/main/providers/s/snowflake-labs .
It'd be "only missing" registering its public key there.

**Comment by tthompson-figma** [comment]
Right, a new publishing step isn't required, just GPG public key submission. OpenTofu doesn't need any special support from the provider otherwise.

If this sounds feasible, you can follow this link to submit it: [Submit new Provider Signing Key](https://github.com/opentofu/registry/issues/new?assignees=&labels=provider-key%2Csubmission&projects=&template=provider_key.yml&title=Provider+Key%3A+)

If you don't have access to the public key anymore, you can retrieve it from the Terraform registry:

```bash
curl 'https://registry.terraform.io/v1/providers/snowflake-labs/snowflake/0.83.1/download/linux/amd64' | jq --raw-output '.signing_keys | .gpg_public_keys | .[0] | .ascii_armor'
```

**Note:** For security reasons, it has to be submitted by the provider author for the OpenTofu registry to accept it.

**Comment by sfc-gh-jcieslak** [comment]
Hey @metaloid2k21 
Is it still a problem? I tested Opentofu with 0.84.0 and newer provider versions (v1.0.0 and v1.0.5), and it worked without any errors.
